Stratis.Bitcoin/Features/RPC/WebHostExtensions.cs

```diff
@@ -10,11 +13,15 @@ public static IWebHostBuilder ForFullNode(this IWebHostBuilder hostBuilder, Full
         {
             hostBuilder.ConfigureServices(s =>
             {
-                s.AddMvcCore(o =>
+                var mvcBuilder = s.AddMvcCore(o =>
                 {
                     o.ModelBinderProviders.Insert(0, new DestinationModelBinder());
                     o.ModelBinderProviders.Insert(0, new MoneyModelBinder());
                 });
+
+                // Include all feature assemblies for action discovery otherwise RPC actions will not execute
```

Comment:
What is exactly the purpose of this code? 
All the controllers/service/feature will exist in the IServiceProvider so they should be available in the controller freamwork as well.

Have a look at Stratis.Bitcoin.Api.MvcBuilderExtensions.AddControllers if that might solve this.

However I like this approach as it will just be consistent with every registered feature.

Reply:
Without that code the RPC actions on the new feature assembles are not discovered and we get an RPC method not found error. I've checked that a number of times.

Reply:
ok cool (next time ad the stackoverflow link in the code)

Reply:
done

Reply:
done